src/pagination/stories/Pagination.component.tsx

```diff
@@ -4,18 +4,12 @@ import {
   Pagination,
   PaginationButton,
   usePaginationState,
+  PaginationInitialState,
 } from "renderless-components";
 
-type TGoto = "next" | "prev" | "last" | "first" | number;
+type TGoto = "nextPage" | "prevPage" | "lastPage" | "firstPage" | number;
```

Comment:
I think we are exporting this type in PaginationButton

Reply:
Yeah !!